bin/integrator

```diff
@@ -18,13 +18,15 @@ if (APPLICATION_VENDOR_DIR !== INTEGRATOR_ROOT_DIR) {
 }
 
 use SprykerSdk\Integrator\Console\ModuleInstallerConsole;
+use SprykerSdk\Integrator\Console\ReleaseGroupInstallerConsole;
 use Symfony\Component\Console\Application;
 
 $application = new Application();
 
 $application->addCommands([
     new ModuleInstallerConsole(),
+    new ReleaseGroupInstallerConsole(),
 ]);
-$application->setDefaultCommand('integrator:manifest:run', true);
+$application->setDefaultCommand('module:manifest:run');
```

Comment:
you shouldn't use the default command if you have more then one. bin/integrator should show the list of commands. btw we have a bug. it makes sense if we use only one command.

Reply:
Instead of all this, please see my comment above
I would rather expect an option flag to set a different source.

Reply:
You are right
Better to delete the `default command` in a separate ticket, because it coud affect the upgrader, because the upgrader trigger the default command